src/NServiceBus.Core.Tests/Config/When_displaying_license_dialog.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace NServiceBus.Core.Tests.Config
+{
+    using System;
+    using NUnit.Framework;
+    using Particular.Licensing;
+
+    [TestFixture(Explicit = true)]
+    public class When_displaying_license_dialog
```

Comment:
Did you use these tests to show the dialog for the screenshots? Because we are trying to get rid of the explicit tests in the core when possible. Could you live with removing those tests and maybe add them as a snippet to this PR?